hotel/hotel/src/main/java/com/hotel/hotel/Dto/GuestDto.java

```diff
@@ -0,0 +1,18 @@
+package com.hotel.hotel.Dto;
+
+import lombok.Data;
+
+
+import java.util.Date;
+
+@Data
+public class GuestDto {
```

Comment:
You don't have to name everything with the suffix dto, but it's ok for now, however, I think a better name would be `GuestRequest` because at the end of the day that's what it is.